### DISCUSSION TITLE
Re-add RPM counter and spinner cleared text at new spinner options

### DISCUSSION THREAD

**Discussion by Ngnhatnam4** [osudroid/osu-droid#338] [Ideas]
I just want to add RPM counter and spinner cleared text back in new spinner options
What do you think about this?Let me know pls

**Comment by Reco1I** [comment]
It was a thing before?

**Reply by Ngnhatnam4** [reply]
Yes this it why I ever want it

**Reply by Ngnhatnam4** [reply]
Too boring spinner loo even the old spinner
